ghzdac/correction.py

```diff
@@ -1165,10 +1188,17 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
                 #decay times:
                 # add to qubit registry the following keys:
                 # settlingAmplitudes=[-0.05]  #relative amplitude
-                # settlingRates = [0.012]    #rate is in GHz, (1/ns)
+                # settlingRates = [0.01]    #rate is in GHz, (1/ns)
                 if np.alen(decayRates):
-                    freqs = 2j*np.pi*freqs
```

Comment:
It actually gets precomputed. The 'precalc' array holds the board response, filter, settling and reflection. It is computed once. Only when reflection or settling is changed, precalc is emptied, and then recalculated, see line 1177:
if np.alen(self.precalc) != nrfft: